src/components/approval-type/ApprovalType.vue

```diff
@@ -8,7 +8,7 @@
         <v-radio-group class="payment-group pt-0" v-model="approvalTypeSelected" @change="radioButtonChanged">
           <!-- COURT ORDER section -->
           <template v-if="!isCourtOrderRadio">
-            <span class="v-label ml-8">{{ getRadioText(ApprovalTypes.VIA_COURT_ORDER) }}</span>
+            <span class="v-label ml-8 mb-4">{{ getRadioText(ApprovalTypes.VIA_COURT_ORDER) }}</span>
```

Comment:
Good catch Sev. In fact, I already have top margin to the component below. I just need to increase that number 👍

Reply:
Oh, on second thought, having extra margins to the component below would make things inconsistent. This is because the component below is the v-text-field. 
In my case above:
- I'm adding the below margin if there is no radio button.
- When we have the radio button, it pushes things down a bit. So If I add the top margin on the component below (text-field), we're going to have a lot of whitespace between the radio button and text-field.

Reply:
For reference:
![margin top](https://user-images.githubusercontent.com/122301442/227056090-3485b0e0-9113-4a82-b20b-0be5582d06da.PNG)
This is when I add the top margin to the component below and if it is a radio button. There is a lot of whitespace which will make things inconsistent.

Reply:
Well, why does having a radio button push things down? Maybe there's a validation element there that can be suppressed -- check the Vuetify API reference for radio buttons. Most components have a a prop to hide that validation element when it's not needed.

Reply:
Good point! I'll look into that.